src/main/java/com/puppycrawl/tools/checkstyle/checks/SuppressWarningsHolder.java

```diff
@@ -441,6 +441,7 @@ private static DetailAST getAcceptableParent(DetailAST child) {
             case TokenTypes.DOT:
             case TokenTypes.ANNOTATION:
             case TokenTypes.MODIFIERS:
+            case TokenTypes.RESOURCE:
```

Comment:
We need to remove all functionality that is logging of violations. 
Please remove line with logging of violation, remove message, all locations of annotations is valid

Reply:
you mean delete getAcceptableParent method and accept all parent type and delete log method right ?

Reply:
@romani Done